src/qt/ApplicationHandler.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <QtCore/QObject>
+#include <QtCore/QTranslator>
+#include <QtQml/qqmlregistration.h>
+
+class ApplicationHandler : public QObject
```

Comment:
Should it be something like TranslationManager?

Reply:
Done